### DISCUSSION TITLE
Licencing questions

### DISCUSSION THREAD

**Discussion by michiiee93** [primefaces/community#674] [PrimeNG] [answered]
Hello,

I tried to contact your support twice by mail, but I didn't receive any answers.
 
I have some questions regarding licensing of PrimeNG. We would really like to use PrimeNG framework + a template, but we are not sure if your licensing allows us to use it.
We are a software company developing standard products for our customers. Important: Our customers operate our software on their systems. We do not run kind of a SaaS or similar.
 
Could you please tell us what licensing we need?
 
We will not be able to always update our software within 6 months on our customer side. So I think the PrimeNG LTS is needed?
And do we need a PrimeNG LTS license for every installation at a customer? Or is one license enough?
Same question for the template.

What happens if customers wait for updates and use an older version of PrimeNG for some time? Is this a problem?
 
Thanks and best regards,
Michael

**Comment by cagataycivici** [comment, marked as answer]
Hi, I've replied via email, could you please confirm?
